Makefile

```diff
@@ -14,7 +15,7 @@ INSTALL_DIR?=$(shell $(REPO_ROOT)/hack/goinstalldir.sh)
 # the output binary name, overridden when cross compiling
 BINARY_NAME=kubetest2-tf
 BINARY_PATH=./kubetest2-tf
-BUILD_FLAGS=-trimpath -ldflags="-buildid= -X=github.com/ppc64le-cloud/kubetest2-plugins/kubetest2-tf/deployer.GitTag=$(COMMIT)"
+BUILD_FLAGS=-trimpath -ldflags="-buildid= -X=github.com/ppc64le-cloud/kubetest2-plugins/kubetest2-tf/deployer.GitTag=$(COMMIT) -X=github.com/ppc64le-cloud/kubetest2-plugins/kubetest2-tf/deployer.TargetProvider=$(PROVIDER)"
```

Comment:
already build mechanism is not that great for this project, lets not add one more to it, lets decide the provider based some flag runtime instead of buildtime.

Reply:
I will the code post this change.

Reply:
Let me check how we can handle this during runtime instead of buildtime.